app/web_conn.go

```diff
@@ -343,8 +348,20 @@ func (wc *WebConn) readPump() {
 	})
 
 	for {
+		msgType, rd, err := wc.WebSocket.NextReader()
+		if err != nil {
+			wc.logSocketErr("websocket.read", err)
+			return
+		}
+
+		var decoder msgDecoder
```

Comment:
Nit: would prefer this to be an anonymous interface to avoid this being declared outside.